lib/excon/connection.rb

```diff
@@ -46,6 +46,7 @@ def proxy=(new_proxy)
     #     @option params [String] :ciphers Only use the specified SSL/TLS cipher suites; use OpenSSL cipher spec format e.g. 'HIGH:!aNULL:!3DES' or 'AES256-SHA:DES-CBC3-SHA'
     #     @option params [String] :proxy Proxy server; e.g. 'http://myproxy.com:8888'
     #     @option params [Fixnum] :retry_limit Set how many times we'll retry a failed request.  (Default 4)
+    #     @option params [Fixnum] :retry_interval Set how long to wait between retries.
```

Comment:
I like that idea for keeping things simple. 
For the exponential backoff, does a interval of 1 second and a rate of 1.5 sound good to you? That will end up with the set of of intervals that I described above 1s, 1.5s, 2.25s, 3.4s, 5.1s, 7.6s,...